util/Migrator/DbScripts/2022-07-11_00_Initial_Organization.sql

```diff
@@ -0,0 +1,547 @@
+IF OBJECT_ID('[dbo].[OrganizationPasswordManager]') IS NOT NULL
+
+BEGIN
+    DROP TABLE [dbo].[OrganizationPasswordManager]
+END
+GO
+
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
```

Comment:
@cscharf do you know why we have `ANSI_NULLS` and `QUOTED_IDENTIFIER` on some migration scripts but not in `dbo`? I'm curious if they are needed.

Reply:
@Hinton , these are not needed, they were an issue once upon a time I believe because those settings got hosted in a prior migration or way some objects were built that required these settings in context of modifications, but those should no longer be necessary... Also, Visual Studio Data Tools automatically adds those when autogenerating delta/update scripts as well, so there may be carryover.

These can be ignored for future migration scripts.

Reply:
More explicitly, yes, please remove these throughout the script.